packages/jest-circus/src/index.ts

```diff
@@ -60,36 +58,15 @@ const _dispatchDescribe = (
   });
   const describeReturn = blockFn();
 
-  // TODO throw in Jest 25
   if (isPromise(describeReturn)) {
-    // eslint-disable-next-line no-console
-    console.log(
-      formatExecError(
-        new ErrorWithStack(
-          chalk.yellow(
-            'Returning a Promise from "describe" is not supported. Tests must be defined synchronously.\n' +
-              'Returning a value from "describe" will fail the test in a future version of Jest.',
-          ),
-          describeFn,
-        ),
-        {rootDir: '', testMatch: []},
-        {noStackTrace: false},
-      ),
+    throw new ErrorWithStack(
```

Comment:
assign `message` to `asyncError` like on lines 46 and 50 for consistency?

Reply:
The todo said 'throw' so I was afraid there might be a difference that makes it important to `throw` :D

Reply:
the error should still be thrown, it's just reusing the error instance we already have instead of creating a new one

Reply:
Looking at the code it seems that was done mostly to not repeat instantiating an error with the `describeFn` stack a lot of times. Since here the error is not even passed to the next `dispatchSync` doesn't seem worth it